### ISSUE TITLE
Accessibility on sortable tables

### ISSUE THREAD

**Issue by mistergone** [cfpb/cf-tables#13]
Tagging @Scotchester, @jimmynotjim, @sebworks, @ascott1 - Please tag anyone else who might be helpful/interested below.

Since our tables are not malformed or incorrect semantically, I see no need for ARIA attributes to be in our code by default for tables themselves.

However, when it comes to the sortable tables and the `.sortable` class, I think we need to take action. For reference, I'm going by [these guidelines](http://www.w3.org/TR/aria-in-html/#first-rule-of-aria-use). There are are two ways I see of addressing sortable accessibility:
1. Change the `.sortable` class from a class attached to a `TH` to a `BUTTON` inside the `TH` which wraps the content (and fills the entire `TH` to create a full interactive area). This adds additional markup, which I may not like, but it follows the "First (and Second) Rule of ARIA use" in the above link, which I am incredibly fond of.
2. Add `role="button"` to the TH. This still adds additional markup and documentation for use, so I don't see it as particularly advantageous over option 1.

There may be more ways of addressing this - I don't claim to be an ARIA/accessibility expert.

So, I'm looking for any advice or support for either of the above methods. I lean towards Option 1, which feels clunky but is ultimately semantic and accessible.

**Please weigh in!!!!**

**Comment by mistergone** [comment]
I realized examples might be helpful.

Option 1 example code:

```
<tr>
  <th>City</th>
  <th><button class="sortable">Distance</button></th>
</tr>
```

Option 2 example code:

```
<tr>
  <th>City</th>
  <th class="sortable" role="button">Distance</th>
</tr>
```

**Comment by cfarm** [comment]
+1 button element

**Comment by contolini** [comment]
:+1: Button element due to [rule #2](http://www.w3.org/TR/aria-in-html/#second-rule-of-aria-use). My only concern would be visual styling of the button.

Thanks for taking this on!

**Comment by mistergone** [comment]
@contolini - I think I will add a class to go with `.sortable`, like `.sortable__button` or something, which strips out our button styles. If anyone has advice on making that easy, please let me know! :octopus: :rocket:

**Comment by Scotchester** [comment]
@mistergone The [`.btn__link` modifier](http://cfpb.github.io/cf-buttons/docs/#button-link) would probably be a good start for you.

**Comment by jimmynotjim** [comment]
Ditto what everyone else has said. Always better to use the native semantics.

**Comment by Scotchester** [comment]
Closed by #16!
